python/graphstorm/distiller/utils.py

```diff
@@ -0,0 +1,49 @@
+"""
+    Copyright 2023 Contributors
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Utils for GNN distillation.
+"""
+
+import torch as th
+
+def to_device(inputs, device='cuda'):
```

Comment:
this seems a common function for others. I suggest to move it to `python/graphstorm/utils.py`

Reply:
this function should also be tested with unit tests

Reply:
removed as suggested by Xiang.